### ISSUE TITLE
[Screen Reader-Social Embedded-Edit Profile]: Talk Back does not convey anything regarding user action.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#16]
"Priority: High

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press ENTER.
11. Navigate to ""Open Navigation Drawer"" button and press ENTER  (or double tap).
12. Navigate to ""My Profile"" and press ENTER  (or double tap) to open.
13. Navigate to  ""Edit Profile"" and press ENTER  (or double tap) to open.
14. Navigate to  ""Upload Photo"" and press ENTER  (or double tap) to open, Select any one option form the two options that is ""Take new photo"" and ""Choose existing photo"".
15. Navigate to ""Edit"" button and press ENTER  (or double tap) to open, Select ""Remove Photo""  and press ENTER  (or double tap) to remove photo.
16. Verify the speech of Talk Back.

Expected
Talk Back should convey proper notification  regarding user action, when user remove, upload the photo as well as create the profile.

Actual
Talk Back does not announce anything regarding user action, when user remove the photo.

Note:
1> Same issue repro when user upload the photo.
2> Same issue repro when user create the profile.

MAS Rule: MAS4.2.12

Impact
User would get confuse and disorient, If Talk Back does not convey any information regarding user action. User will face difficulty to know the same.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
